Reply/app/src/main/res/values/shape.xml

```diff
@@ -31,6 +31,17 @@
         <item name="cornerSize">@dimen/reply_large_component_corner_radius</item>
     </style>
 
+    <style name="ShapeAppearanceOverlay.Reply.Button.Rounded" parent="">
+        <item name="cornerFamily">rounded</item>
+        <item name="cornerSize">50%</item>
+    </style>
+
+    <!--ShapeAppearance overlay for components which can be 'marked' by rounding their top left -->
+    <!--corner - ie. email cards which have been starred-->
+    <style name="ShapeAppearanceOverlay.Reply.MediumComponent.Marked" parent="">
+        <item name="cornerSizeTopLeft">@dimen/reply_small_component_corner_radius</item>
+    </style>
```

Comment:
delete `ShapeAppearanceOverlay.Reply.Button.Rounded` and `ShapeAppearanceOverlay.Reply.MediumComponent.Marked`? looks like they're unused